src/utils/vue-dts.ts

```diff
@@ -30,22 +30,22 @@ export async function getVueDeclarations(
     return;
   }
 
-  const ts = await import("typescript").then((r) => r.default || r);
+  // Inside vue-tsc, `require` is used instead of `import`. In order to override `ts.sys`, it is necessary to import it in the same way as vue-tsc for them to refer to the same file.
+  const ts =
+    require("typescript") as typeof import("typescript/lib/tsserverlibrary");
```

Comment:
We need to use `createRequire` from `node:module` to create a new require this require is not available in strict `.mjs`

Reply:
fix in 6a29adf172aa62934242a17fb3fe86e2b1064f58

Reply:
Hi @pi0 ! I have fixed it. Please take another look. Thank you.